integrations/circleci/README.md

```diff
@@ -38,7 +38,7 @@ workflows:
 version: 2.1
 
 orbs:
-  salus: federacy/salus@2.5.0
+  salus: federacy/salus@2.5.1
```

Comment:
I think this will break if we bump the orbs reference if we bump before it's live.